From 208f34cc4de09ff0a91e99e9e7bba66569a98faa Mon Sep 17 00:00:00 2001
From: Xie Haocheng <haocheng.xie@amd.com>
Date: Tue, 25 Apr 2023 11:01:17 +0800
Subject: [PATCH 0414/1646] [Backport] Revert "kabi: Fix kabi breakage caused
 by commit d5616bac7ada."

amd inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I6XNL2

Reference: https://gitee.com/openeuler/kernel/commit/fb465f689c9ffa8b6d1dea088ad6e47ee36494ab

-------------------------------------------------

This reverts commit a9cbff64a895f2dc1834df2c2178586fafd7d212.

This patch could introduce build warnings, should be reverted.
The build warning messages:
WARNING: modpost: EXPORT symbol "__SCT__perf_lopwr_cb" [vmlinux] version generation failed, symbol will not be versioned.
    WARNING: modpost: EXPORT symbol "__SCT__perf_lopwr_cb" [vmlinux] version generation failed, symbol will not be versioned.

Signed-off-by: Xie Haocheng <haocheng.xie@amd.com>
Signed-off-by: Lin Yujun <linyujun809@huawei.com>
---
 arch/x86/include/asm/perf_event.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/x86/include/asm/perf_event.h b/arch/x86/include/asm/perf_event.h
index 8d684a0fc6f1..c8fa0c981435 100644
--- a/arch/x86/include/asm/perf_event.h
+++ b/arch/x86/include/asm/perf_event.h
@@ -2,9 +2,7 @@
 #ifndef _ASM_X86_PERF_EVENT_H
 #define _ASM_X86_PERF_EVENT_H
 
-#ifndef __GENKSYMS__
 #include <linux/static_call.h>
-#endif
 
 /*
  * Performance event hw details:
-- 
2.17.1

